02-household-queries/requirements.in

```diff
@@ -5,6 +5,7 @@
 beautifulsoup4
 chainlit
 chromadb
+cohere
```

Comment:
Reminder to run `pip-compile requirements.in` to update `requirements.txt`

Reply:
Read this comment, thought "ah right", and then pushed merge 😂  -- I'll update on the base branch now before merging.